docs/source/index.rst

```diff
@@ -23,7 +23,7 @@ Cheatsheet
 
 - `NoSkill modding guide (highly recommended to check out)
   <https://noskill.gitbook.io/titanfall2/>`_
-- `Tools for modding <tools.md>`_
+- `Tools for modding </guides/tools/index.html>`_
```

Comment:
Link is still broken in the test build. This should work (probably)
```suggestion
- :doc:`Tools for modding <./guides/tools/index>` 
```

Reply:
I'll just commit and see if it works hehe

Reply:
I did it for you now <3